test/e2e/apps/rc.go

```diff
@@ -45,7 +46,7 @@ var _ = SIGDescribe("ReplicationController", func() {
 		// requires private images
 		framework.SkipUnlessProviderIs("gce", "gke")
 
-		TestReplicationControllerServeImageOrFail(f, "private", "gcr.io/k8s-authenticated-test/serve_hostname:v1.4")
+		TestReplicationControllerServeImageOrFail(f, "private", imageutils.GetE2EImage(imageutils.ServeHostname))
```

Comment:
then we may need to push `serve_hostname` image to `gcr.io/k8s-authenticated-test` repo as well for all the architectures or do you have any other way to tackle this?